src/ComponentFolders/AboutPage/About.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import aboutImg from './Images/man.jpg';
+
+function About() {
+  return (
+    <div className="flex flex-col w-screen h-screen font-poppins text-black tracking-[-0.01em]">
+      <div className="flex flex-grow bg-white flex-col lg:mb-[0%] mb-[5%] ml-[10%] mt-[5%] mr-[8%]">
+        <h2 className="lg:text-5xl text-3xl">HEALING!</h2>
+        <h3 className="lg:text-3xl text-xl text-black text-opacity-50">
+          some cool liner !
+        </h3>
+        <div className="md:text-lg text-sm leading-6 mt-[5%]">
+          At Healing, we believe there is a better way to do things. A more
+          valuable way where customers are earned rather than bought. We&apos;re
+          obsessively passionate about it, and our mission is to help people
+          achieve it. We focus on search engine optimization. It&apos;s one of
+          the least understood and least transparent aspects of great marketing,
+          and we see that as an opportunity. We&apos;re excited to simplify SEO
+          for everyone through our software, education, and community.
+        </div>
+      </div>
+      <div className="flex flex-grow flex-col lg:flex-row bg-blue-100 w-full ">
```

Comment:
can you change background color to "bg-cyan-50" so that it would match the navbar :D